File: csharp/LinkedList.cs
using System;

public class Node
{
    public int Value;
    public Node? Next;

    public Node(int value)
    {
        Value = value;
        Next = null;
    }
}

public class LinkedList
{
    private Node? head;

    public void Append(int value)
    {
        Node newNode = new Node(value);

        if (head == null)
        {
            head = newNode;
            return;
        }

        Node current = head;
        while (current.Next != null)
        {
            current = current.Next;
        }

        current.Next = newNode;
    }

    public void PrintList()
    {
        Node? current = head;
        while (current != null)
        {
            Console.Write($"{current.Value} → ");
            current = current.Next;
        }
        Console.WriteLine("null");
    }
}

public static class LinkedListDemo
{
    public static void RunLinkedListDemo()
    {
        Console.WriteLine("\n🔗 Linked List Demo");
        var list = new LinkedList();
        list.Append(1);
        list.Append(2);
        list.Append(3);
        list.PrintList();
    }
}

File: python/linkedlist.py
class Node:
    def __init__(self, value):
        self.value = value
        self.next = None


class LinkedList:
    def __init__(self):
        self.head = None

    def append(self, value):
        new_node = Node(value)

        if self.head is None:
            self.head = new_node
            return

        current = self.head
        while current.next:
            current = current.next

        current.next = new_node

    def print_list(self):
        current = self.head
        while current:
            print(current.value, end=" → ")
            current = current.next
        print("None")


def run_linked_list_demo():
    print("\n🔗 Linked List Demo")
    ll = LinkedList()
    ll.append(1)
    ll.append(2)
    ll.append(3)
    ll.print_list()
